cmd/server/assets/codestatus/index.html

```diff
@@ -21,16 +21,17 @@ <h1>Verification code status</h1>
       Use an identifier to check the status of a code previously shared with your patient.
     </p>
 
-    <div class="card mb-3">
+    <div class="card mb-3" style="max-width: 30rem">
```

Comment:
Why max this?

Reply:
the list looks kind of weird as it gets huge with the whitespace. Originally I had the button floating right, which looked bad if it was too wide. I'd rather center the card div or something.

That said I'm open to aesthetic tweaks